pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[tool.black]
+line-length = 88
```

Comment:
Check that this is consistent between black and flake8

Reply:
See above comment on the .flake8 review